source/normalize-arguments.ts

```diff
@@ -405,5 +407,8 @@ export const normalizeRequestArguments = async (options: NormalizedOptions): Pro
 
 	delete options.timeout;
 
+	// `http-cache-semantics` check this
```

Comment:
`check` => `checks`

Reply:
Good spot! Didn't know that `semantics` is uncountable.

Reply:
You mean not plural? yeah.